### ISSUE TITLE
Redirection issue with Wordpress REST API and hide default language in url set to true.

### ISSUE THREAD

**Issue by herrvigg** [qtranslate/qtranslate-xt#427]
<a href="https://github.com/esiao"><img src="https://avatars3.githubusercontent.com/u/1942801?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [esiao](https://github.com/esiao)**
_Friday Sep 02, 2016 at 15:37 GMT_
_Originally opened as https://github.com/qTranslate-Team/qtranslate-x/issues/427_

----

When trying to fetch content with the REST API endpoint. (http://v2.wp-api.org/)
There's a redirection happening, `site.local.dev/en/wp-json/wp/v2/` is redirected to `site.local.dev/wp-json/wp/v2/` and two errors are outputed:

```
( ! ) Notice: Undefined index: wp-path in [...]/plugins/qtranslate-x/qtranslate_core.php on line 953

( ! ) Warning: Cannot modify header information - headers already sent by (output started at [...]/plugins/qtranslate-x/qtranslate_core.php:953) in [...]/wp-includes/pluggable.php on line 1174
```

It's not possible to use the Rest API with this setting set to true, if it's set to false there's no redirection and it's working as expected.

**Comment by herrvigg** [comment]
<a href="https://github.com/sandrodz"><img src="https://avatars1.githubusercontent.com/u/8479569?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [sandrodz](https://github.com/sandrodz)**
_Friday Nov 25, 2016 at 14:01 GMT_

----

I see same error on `wp-admin` url, using bedrock.

**Comment by herrvigg** [comment]
<a href="https://github.com/carlosreig"><img src="https://avatars2.githubusercontent.com/u/2478462?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [carlosreig](https://github.com/carlosreig)**
_Friday Oct 13, 2017 at 09:28 GMT_

----

Same error `wp-admin` URL using bedrock

**Comment by herrvigg** [comment]
Fix released in qTranslate-XT [3.5.3](https://github.com/qtranslate/qtranslate-xt/releases/tag/3.5.3).
